bouter/embedded/__init__.py

```diff
@@ -10,8 +10,9 @@ def __init__(self, *args, continue_curvature=5, **kwargs):
         super().__init__(*args, **kwargs)
 
         if continue_curvature is not None:
-            utilities.fill_out_segments(
-                self.tail_points_matrix, continue_curvature=continue_curvature
+            self.tail_points_matrix, _ = utilities.fill_out_segments(
```

Comment:
you are discarding the information which tail segments were filled out, did we not agree to update the h5 dataframe with the reconstructed tail and one column extra to indicate where the tracking stopped?

Reply:
Oh right